src/network/networkd-link.c

```diff
@@ -1161,6 +1162,16 @@ static int link_set_flags(Link *link) {
                 SET_FLAG(ifi_flags, IFF_ALLMULTI, link->network->allmulticast);
         }
 
+        if (link->network->promiscuous >= 0) {
+                ifi_change |= IFF_PROMISC;
+                SET_FLAG(ifi_flags, IFF_PROMISC, link->network->promiscuous);
+        }
+
+        if (link->network->trailers >= 0) {
+                ifi_change |= IFF_NOTRAILERS;
+                SET_FLAG(ifi_flags, IFF_NOTRAILERS, !link->network->trailers);
+        }
```

Comment:
Humm, what? what is this supposed to do? What even is a "trailer"? as far as i could find, IFF_NOTRAILERS is a NOP on linux. See this discussion:

http://lists.landley.net/pipermail/toybox-landley.net/2013-November/001462.html

Please, let's apply some restraint here. Exposing useful options, yes. But NOPs? No!